Backend/create-tables.sql

```diff
@@ -1,30 +1,34 @@
 CREATE TABLE Courses (
-    course_id varchar(100) PRIMARY KEY,
+    course_id varchar(100),
+    PRIMARY KEY(course_id, quarter),
```

Comment:
can this line be before you declare what quarter is?

Reply:
I dont think it can be, I'll change that in my next pull request